### PR TITLE
Minor Fixes on Beginner

### PR DESCRIPTION
**Pull request by EpicSupermind**
http:// -> https://
osu! -> *osu!*
Feel free to review on anything, i mean everything i missed.

### CODE REVIEW

File: wiki/Beginner/en.md [hunk 1]
```diff
@@ -25,7 +27,7 @@ Mouse play tends to struggle with:
...
 
-If you are going to use a tablet, you should try and pick an area on the tablet that you can comfortably play with. If you are going to buy a tablet to play osu! with, you should check out the [Tablet purchase](wiki/List_of_Guides/Tablet_purchase "Tablet purchase by Daru") guide. Additionally, buy some spare nibs, you never know when you will need another one since they tend to wear out quickly, especially in highly active play. While you can use the tablet itself to tap the circles, most players prefer using a keyboard which we'll cover a bit later on. Tablet play tends to excel at:
+If you are going to use a tablet, you should try and pick an area on the tablet that you can comfortably play with. If you are going to buy a tablet to play *osu!* with, you should check out the [Tablet purchase](wiki/List_of_Guides/Tablet_purchase "Tablet purchase by Daru") guide. Additionally, buy some spare nibs, you never know when you will need another one since they tend to wear out quickly, especially in highly active play. While you can use the tablet itself to tap the circles, most players prefer using a keyboard which we'll cover a bit later on. Tablet play tends to excel at:
```
Comment (Shadowtail117) [inline wiki/Beginner/en.md]: "Additionally, buy some spare nibs, _as_ you never know..." OR
"Additionally, buy some spare nibs; you never know..."

File: wiki/Beginner/en.md [hunk 2]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+One of the most important factors when playing *osu!* is your timing, much like other rhythm games! In this sense, if you've listened to a lot of music, it should be fairly easy to pick up on. The goal of accuracy and timing is to tap the notes at the correct time as precisely as possible.
 
 Each object in the game has an approach circle; a large ring that slowly approaches the hit circle itself. When the ring reaches the circle, it's time to tap it! If you pay attention to them carefully, it will help you recognize whether some notes are on half-beats or full beats, or whether you need to wait for the next note.
 
 Music works in beats and a rhythm. When you start a map (or even before you play) pay attention to the how the song sounds, and you can get a pretty good idea on the melody that you will be hitting circles to. Getting along with the music more closely will help you out in achieving better timing and coordination over it.
```
Comment (Shadowtail117) [inline wiki/Beginner/en.md]: "Music works in beats and rhythms. When you start a map (or even before you play), pay attention..."

(Get rid of the "a" and pluralize rhythms; add a comma after the parentheses.)

File: wiki/Beginner/en.md [hunk 3]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+In a point-and-click game like *osu!*, aiming is also another important factor. Being able to get the notes at the right time is pretty pointless without being able to get your cursor to the right location. In this case, the most effective way to improve in this sector is to simply play the game. It also helps to play harder and harder maps, since the spacing between notes will likely get bigger, making you move around more and tap the notes from wider distances.
 
-The best way you can get practice for aiming without osu! is using your computer with the input device. Don't be afraid to practice by clicking around files on your computer if you're a mouse user, or to draw some art with your tablet. The added coordination will help you wonders later on!
+The best way you can get practice for aiming without *osu!* is using your computer with the input device. Don't be afraid to practice by clicking around files on your computer if you're a mouse user, or to draw some art with your tablet. The added coordination will help you wonders later on!
```
Comment (Shadowtail117) [inline wiki/Beginner/en.md]: "The best way you can practice for aiming without *osu!* is _by_ using your computer with your chosen input device."

File: wiki/Beginner/en.md [hunk 4]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
 ### Speed
 
 Eventually, when you get to harder difficulties, you will need to click the notes more rapidly. Improving in this fact is actually quite simple; all you need to do is push yourself into clicking or tapping the keyboard faster. You can play games that require you to hit input as fast as possible, like Cookie Clicker or other button pushing games (they are very easy to find online if you just Google it).
```
Comment (Shadowtail117) [inline wiki/Beginner/en.md]: "Improving in this is actually quite simple..." (Get rid of "this")
"You can play games that require you to input as fast as possible..." (Get rid of "hit")

File: wiki/Beginner/en.md [hunk 5]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
 ### Practice
 
-Another heavy topic, is the fact that osu! is a game with an infinitely rising difficulty curve, meaning there is no "winning" the game. Instead, you'll be focused on improving yourself constantly. Instead of playing one map over and over, you will need to eventually move onto other maps once you start outclassing their difficulty.
+Another heavy topic, is the fact that *osu!* is a game with an infinitely rising difficulty curve, meaning there is no "winning" the game. Instead, you'll be focused on improving yourself constantly. Instead of playing one map over and over, you will need to eventually move onto other maps once you start outclassing their difficulty.
```
Comment (Shadowtail117) [inline wiki/Beginner/en.md]: "Another heavy topic is the fact..." (Get rid of the comma)

File: wiki/Beginner/en.md [hunk 6]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
-Don't be afraid to practise something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions (and more) that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
+Don't be afraid to practice something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
```
Comment (Shadowtail117) [inline wiki/Beginner/en.md]: Practise is the British verb form of the word (as opposed to practice being the noun). Therefore, you don't need to alter them.

Comment (EpicSupermind) [inline wiki/Beginner/en.md]: oh okay, i'm American xD

Comment (Shadowtail117) [inline wiki/Beginner/en.md]: Same, I bet there's a couple British flaws in what I've written too :P

File: wiki/Beginner/en.md (hunk 3 above)
Comment (oralekin) [inline wiki/Beginner/en.md]: `without playing` would be better imo, it sounds like osu! is a physical thing...

File: wiki/Beginner/en.md [hunk 7]
```diff
@@ -25,7 +27,7 @@ Mouse play tends to struggle with:
...
 
-If you are going to use a tablet, you should try and pick an area on the tablet that you can comfortably play with. If you are going to buy a tablet to play osu! with, you should check out the [Tablet purchase](wiki/List_of_Guides/Tablet_purchase "Tablet purchase by Daru") guide. Additionally, buy some spare nibs, you never know when you will need another one since they tend to wear out quickly, especially in highly active play. While you can use the tablet itself to tap the circles, most players prefer using a keyboard which we'll cover a bit later on. Tablet play tends to excel at:
+If you are going to use a tablet, you should try and pick an area on the tablet that you can comfortably play with. If you are going to buy a tablet to play *osu!* with, you should check out the [Tablet purchase](wiki/List_of_Guides/Tablet_purchase "Tablet purchase by Daru") guide. Additionally, buy some spare nibs, as you never know when you will need another one since they tend to wear out quickly, especially in highly active play. While you can use the tablet itself to tap the circles, most players prefer using a keyboard which we'll cover a bit later on. Tablet play tends to excel at:
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]:
```
[Tablet purchase](/wiki/List_of_Guides/Tablet_purchase "Tablet purchase by Daru")
```
(missing slash `/` between `se](` and `wiki`)

File: wiki/Beginner/en.md [hunk 8]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
--   [Sean Paul - She Doesn't Mind (Daniel Ngo Remix)](http://osu.ppy.sh/b/158917) on Easy (1.08 Stars)
--   [Anamanaguchi - This is the End](http://osu.ppy.sh/b/74686) on Average Joe (1.09 Stars)
--   [Infected Mushroom - The Pretender](http://osu.ppy.sh/b/240979) on Easy (1.10 Stars)
--   [Do As Infinity - Fukai Mori](http://osu.ppy.sh/b/49478) on Easy Dan (1.11 Stars)
--   [Nishiura Tomohito - Shop](http://osu.ppy.sh/b/404645) on Easy (1.13 Stars)
--   [Angela Chang - Aurora](http://osu.ppy.sh/b/397975) on Easy (1.14 Stars)
-
-Novice
-------
+-   [GReeeeN - Ai Uta (Taiko Cut)](https://osu.ppy.sh/b/57281) on Kantan (0.83 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `GReeeeN - Ai Uta (Taiko Cut) (arken1015) \[Kantan\]` as link text

File: wiki/Beginner/en.md [hunk 9]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [TOKINE - Soko ni Aru Mono](https://osu.ppy.sh/b/67008) on Easy (0.99 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `TOKINE - Soko ni Aru Mono (xierbaliti) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 10]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Rammstein - Mein erz brennt (Piano Instrumental)](https://osu.ppy.sh/b/250553) on Easy (1.00 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Rammstein - Mein Herz brennt (Piano Instrumental) (Sieg) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 11]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [fripSide - a little christmas time](https://osu.ppy.sh/b/191008) on Easy (1.03 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `fripSide - a little christmas time (Darksonic) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 12]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Yoko Shimomura - A Twinkle in the Sky](https://osu.ppy.sh/b/220650) on Easy (1.05 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Yoko Shimomura - A Twinkle in the Sky (MoodyRPG) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 13]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Sean Paul - She Doesn't Mind (Daniel Ngo Remix)](https://osu.ppy.sh/b/158917) on Easy (1.08 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Sean Paul - She Doesn't Mind (Daniel Ngo Remix) (BounceBabe) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 14]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Anamanaguchi - This is the End](https://osu.ppy.sh/b/74686) on Average Joe (1.09 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Anamanaguchi - This is the End (Kurai) \[Average Joe\]` as link text

File: wiki/Beginner/en.md [hunk 15]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Infected Mushroom - The Pretender](https://osu.ppy.sh/b/240979) on Easy (1.10 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Infected Mushroom - The Pretender (Nemis) \[Easy\]`

File: wiki/Beginner/en.md [hunk 16]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Do As Infinity - Fukai Mori](https://osu.ppy.sh/b/49478) on Easy Dan (1.11 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Do As Infinity - Fukai Mori (Andrea) \[Easy Dan\]` as link text

File: wiki/Beginner/en.md [hunk 17]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Nishiura Tomohito - Shop](https://osu.ppy.sh/b/404645) on Easy (1.13 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Nishiura Tomohito - Shop (samplefish) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 18]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Angela Chang - Aurora](https://osu.ppy.sh/b/397975) on Easy (1.14 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Angela Chang - Aurora (Pasha_Khvan97) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 19]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
 After you play a bit, try these maps out.
 
--   [Mutsuhiko Izumi - Russian Snowy Dance](http://osu.ppy.sh/b/124348) on Easy (1.21 Stars)
--   [goreshit - o'er the flood](http://osu.ppy.sh/b/177139) on Easy (1.22 Stars)
--   [Hatsune Miku - Electric Love (Short Ver.)](http://osu.ppy.sh/b/141749) on Easy. (1.25 Stars)
--   [DJ YOSHITAKA feat. Hoshino Kanako - MAX LOVE](http://osu.ppy.sh/b/186601) on Easy (1.28 Stars)
--   [Akcent feat. Edward Maya - That's My Name](http://osu.ppy.sh/b/132838) on Easy (1.28 Stars)
--   [Hatsune Miku - Do-re-mi-fa Rondo](http://osu.ppy.sh/b/163457) on Easy (1.28 Stars)
+-   [Mutsuhiko Izumi - Russian Snowy Dance](https://osu.ppy.sh/b/124348) on Easy (1.21 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Mutsuhiko Izumi - Russian Snowy Dance (Chocopikel) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 20]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [goreshit - o'er the flood](https://osu.ppy.sh/b/177139) on Easy (1.22 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `goreshit - o'er the flood (grumd) \[Easy\]` as link text

File: wiki/Beginner/en.md (hunk 8 above)
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: for this and the following, remove the `on xxxxxxx` part

File: wiki/Beginner/en.md [hunk 21]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Hatsune Miku - Electric Love (Short Ver.)](https://osu.ppy.sh/b/141749) on Easy. (1.25 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Hatsune Miku - Electric Love (Short Ver.) (KayeSara) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 22]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [DJ YOSHITAKA feat. Hoshino Kanako - MAX LOVE](https://osu.ppy.sh/b/186601) on Easy (1.28 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `DJ YOSHITAKA feat. Hoshino Kanako - MAX LOVE (Mythol) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 23]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Akcent feat. Edward Maya - That's My Name](https://osu.ppy.sh/b/132838) on Easy (1.28 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Akcent feat. Edward Maya - That's My Name (Barney Stinson) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 24]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Hatsune Miku - Do-re-mi-fa Rondo](https://osu.ppy.sh/b/163457) on Easy (1.28 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Hatsune Miku - Do-re-mi-fa Rondo (Sylphi) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 25]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+### Challenge
 
 Want to push yourself and take on a challenge? Try these!
 
--   [DJ Fresh - Gold Dust](http://osu.ppy.sh/b/96042) on Easy (1.32 Stars)
--   [Kyary Pamyu Pamyu - Ninjya Re Bang Bang](http://osu.ppy.sh/b/230174) on Easy (1.36 Stars)
--   [Carly Rae Jepsen - Call Me Maybe (Nightcore Mix)](http://osu.ppy.sh/b/172736) on Easy (1.36 Stars)
--   [Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix)](http://osu.ppy.sh/b/368307) on Easy (1.45 Stars)
--   [Gentle Stick X M2U - Hades in the Heaven](http://osu.ppy.sh/b/318994) on Easy (1.46 Stars)
+-   [DJ Fresh - Gold Dust](https://osu.ppy.sh/b/96042) on Easy (1.32 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `DJ Fresh - Gold Dust (galvenize) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 26]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Kyary Pamyu Pamyu - Ninjya Re Bang Bang](https://osu.ppy.sh/b/230174) on Easy (1.36 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Kyary Pamyu Pamyu - Ninjya Re Bang Bang (IchiGUN) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 27]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Carly Rae Jepsen - Call Me Maybe (Nightcore Mix)](https://osu.ppy.sh/b/172736) on Easy (1.36 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Carly Rae Jepsen - Call Me Maybe (Nightcore Mix) (Bakari) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 28]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix)](https://osu.ppy.sh/b/368307) on Easy (1.45 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix) (Ami) \[Easy\]` as link text

File: wiki/Beginner/en.md [hunk 29]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+-   [Gentle Stick X M2U - Hades in the Heaven](https://osu.ppy.sh/b/318994) on Easy (1.46 Stars)
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: use `Gentle Stick X M2U - Hades in the Heaven (AngelHoney) \[Easy\]` for link text

File: wiki/Beginner/en.md [hunk 30]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
-Going Beyond
-============
+## Going Beyond
 
-Once you have exhausted the difficulty of the beatmaps listed here, don't stop there. Keep on going, and find some maps of a higher difficulty rating. Try even searching for beatmaps based on [difficulty suggestion](http://osu.ppy.sh/p/beatmaplist?s=3&o=-1&m=0&r=0&g=0&la=0&ra), as it will help you get an idea of where you need to go next. Don't be afraid to ask other users too about how you can improve, most often they will be nice and share you a pretty fun map to play that they also enjoyed, where their osu! world adventure begun. Most importantly, have fun!
+Once you have exhausted the difficulty of the beatmaps listed here, don't stop there. Keep on going, and find some maps of a higher difficulty rating. Try even searching for beatmaps based on [difficulty suggestion](https://osu.ppy.sh/p/beatmaplist?s=3&o=-1&m=0&r=0&g=0&la=0&ra), as it will help you get an idea of where you need to go next. Don't be afraid to ask other users too about how you can improve, most often they will be nice and share you a pretty fun map to play that they also enjoyed, where their *osu!* world adventure begun. Most importantly, have fun!
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: fix this sentence
```
Don't be afraid to ask other users about improving, most often they will be nice and share you a pretty fun map to play that they also enjoyed, where their *osu!* world adventure begun. Lastly, but most importantly, have fun!
```
(and this is why I break all sentences down individually :/)

File: wiki/Beginner/en.md [hunk 31]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
+In a point-and-click game like *osu!*, aiming is also another important factor. Being able to get the notes at the right time is pretty pointless without being able to get your cursor to the right location. In this case, the most effective way to improve in this sector is to simply play the game. It also helps to play harder and harder maps, since the spacing between notes will likely get bigger, making you move around more and tap the notes from wider distances.
 
-The best way you can get practice for aiming without osu! is using your computer with the input device. Don't be afraid to practice by clicking around files on your computer if you're a mouse user, or to draw some art with your tablet. The added coordination will help you wonders later on!
+The best way you can practice for aiming without playing *osu!* is using your computer with your input device of choice. Don't be afraid to practice by clicking around files on your computer if you're a mouse user, or to draw some art with your tablet. The added coordination will help you wonders later on!
```
Comment (TPGPL) [inline wiki/Beginner/en.md]: practice for aiming -> practice aiming

Comment (TPGPL) [inline wiki/Beginner/en.md]: with your input device of choice -> with input device of your own choice

File: wiki/Beginner/en.md [hunk 32]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
-Don't be afraid to practise something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions (and more) that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
+Don't be afraid to practise something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
 
 Getting More out of Gameplay
```
Comment (TPGPL) [inline wiki/Beginner/en.md]: More -> more?

File: wiki/Beginner/en.md [hunk 33]
```diff
@@ -1,4 +1,6 @@
-Welcome to the osu! beginner tutorial. The world of osu! can sometimes be a very competitive place. Whether you are looking for an extra push to help lift off your career, wanting to be competitive in osu!, or simply hoping to improve yourself to enjoy the game more, this is for you! This guide will span everything you need to get off the ground and soar around the world of osu!.
+# Beginner
+
+Welcome to the *osu!* beginner tutorial. The world of *osu!* can sometimes be a very competitive place. Whether you are looking for an extra push to help lift off your career, wanting to be competitive in *osu!*, or simply hoping to improve yourself to enjoy the game more, this is for you! This guide will span everything you need to get off the ground and soar around the world of *osu!*.
 
 Types of Input
 --------------
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: add `_See also [Play Styles](/wiki/Play_Styles)._`

File: wiki/Beginner/en.md [hunk 34]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
-Don't be afraid to practise something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions (and more) that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
+Don't be afraid to practise something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
 
-Getting More out of Gameplay
+Getting more out of Gameplay
 ----------------------------
 
 ### Global Offset
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: add `_See also [How to use the Offset Wizard](/wiki/List_of_Guides/How_to_use_the_Offset_Wizard)._`

File: wiki/Beginner/en.md [hunk 35]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
-In the Offset Wizard, which you can access from the Settings menu, a sample song will play with a metronome beat on top. Adjust the offset gradually using the up/down arrow keys or your mouse wheel until the beats of the metronome match when the bars cross the centre line.
+In the [Offset Wizard](/wiki/Options/Universal_Offset), which you can access from the [Settings](/wiki/Options/) menu, a sample song will play with a metronome beat on top. Adjust the offset gradually using the up/down arrow keys or your mouse wheel until the beats of the metronome match when the bars cross the centre line.
 
 When you are playing a beatmap you will notice the Hit Error graph at the bottom of the screen. Hitting a note causes a white line to appear somewhere along it depending on how accurate you were. If all of your hits are to the left of the centre, try increasing your global offset slightly. If they are all on the right side, try decreasing it.
 
 Adjust your global offset until you are happy with it. Getting the right value will help to increase your overall accuracy!
 
 ### Raw Input
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: add `_See also [Options](/wiki/Options/#mouse)._`

File: wiki/Beginner/en.md [hunk 36]
```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
...
 
-A commonly overlooked feature of the game is the Raw Input function. This allows for direct reading of your input device into osu! itself. You may use this too if you don't like how Windows handles your mouse cursor, or especially recommended if you use a tablet. Give it a try and play a few maps with it, you might find yourself a lot more comfortable with it!
+A commonly overlooked feature of the game is the Raw Input function. This allows for direct reading of your input device into *osu!* itself. You may use this too if you don't like how Windows handles your mouse cursor, or especially recommended if you use a tablet. Give it a try and play a few maps with it, you might find yourself a lot more comfortable with it!
 
 ### Skins
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: add `_See also [Skinning](/wiki/Skinning/)._`

File: wiki/Beginner/en.md [hunk 37]
```diff
@@ -1,5 +1,7 @@
 # Beginner
 
+*See also [Play Styles](/wiki/Play_Styles).*
```
Comment (MegaApplePi) [inline wiki/Beginner/en.md]: wrong spot, move under line 8 (Types of Input section)

**Pull request merged**